src/status_im/contexts/communities/overview/view.cljs

```diff
@@ -369,18 +370,24 @@
 
 (defn- community-fetching-placeholder
   [id]
-  (let [fetching? (rf/sub [:communities/fetching-community id])]
+  (let [theme      (quo.theme/use-theme)
+        margin-top (safe-area/get-top)
+        fetching?  (rf/sub [:communities/fetching-community id])]
     [rn/view
-     {:style               style/fetching-placeholder
+     {:style               (style/fetching-placeholder margin-top)
```

Comment:
small nit 🔬 
maybe we can rename `margin-top` to `top-inset?

Reply:
done, thank you